### PR TITLE
Eswe 765 setup the gradle framework

### PR DESCRIPTION
**Pull request by sacintha**

### CODE REVIEW

File: src/main/resources/migration/V1_1__start.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+drop table IF EXISTS ciag_profile CASCADE;
```
Comment (theDustRoom) [inline src/main/resources/migration/V1_1__start.sql]: We have reference to CIAG here

**Review by theDustRoom — CHANGES_REQUESTED** [review]
We still have a reference to CIAG that needs to be changed

File: src/main/resources/migration/V1_1__start.sql (hunk 1 above)
Comment (sacintha) [reply src/main/resources/migration/V1_1__start.sql]: Fixed

Comment (theDustRoom) [reply src/main/resources/migration/V1_1__start.sql]: Thanks Sacintha

**Review by theDustRoom — APPROVED** [review]
Builds and tests ok

**Review by theDustRoom — APPROVED** [review]
Builds and tests ok

**Pull request merged**